doc/content/verification_validation_examples/cad_workflow/cad_workflow.md

```diff
@@ -128,21 +129,21 @@ The results of this simulation on the finest mesh are shown in [temperatures], [
 
 !media figures/cad_workflow_tritium_production.png
   id=tritium_production
-  caption=Tritium production rate density result from [!cite](Eltawila2024PBNC)
+  caption=Tritium production rate density result from [!cite](Eltawila2024PBNC).
   style=display:block;margin-left:auto;margin-right:auto;width:130%;
 
 !col-end!
 !row-end!
 
 &nbsp;
 
-!table id=results caption=Results summary
-| Parameter (Units) | Value |
-| :- | :- |
-| Armor Max. Temp. (K)         | $1062.4$                   |
-| First Wall Max. Temp. (K)    | $1057.6$                   |
-| Breeder Max. Temp. (K)       | $987.4$                    |
-| Heat Source (W)              | $2.44 \times 10^5 \pm 3 \times 10^3$   |
-| Tritium Production (atoms/s) | $4.70 \times 10^{13} \pm 8 \times 10^{11}$ |
+!table id=results caption=Summary of the results from the coupled simulation published in [!cite](Eltawila2024PBNC).
+| Parameter (Units) | Value | Units |
+| :- | :- | :- |
+| Armor Max. Temp.          | $1062.4$                                   | `[K]`      |
+| First Wall Max. Temp.     | $1057.6$                                   | `[K]`      |
+| Breeder Max. Temp.        | $987.4$                                    | `[K]`      |
+| Heat Source               | $2.44 \times 10^5 \pm 3 \times 10^3$       | `[W]`      |
+| Tritium Production.       | $4.70 \times 10^{13} \pm 8 \times 10^{11}$ | `[atoms/s]`|
```

Comment:
```suggestion
| Armor Max. Temperature          | $1062.4$                                   | `[K]`      |
| First Wall Max. Temperature     | $1057.6$                                   | `[K]`      |
| Breeder Max. Temperature        | $987.4$                                    | `[K]`      |
| Heat Source               | $2.44 \times 10^5 \pm 3 \times 10^3$       | `[W]`      |
| Tritium Production       | $4.70 \times 10^{13} \pm 8 \times 10^{11}$ | `[atoms/s]` |
```
The | appears in the documentation. I am adding a space to avoid this.